docs/HRANA_3_SPEC.md

```diff
@@ -1026,6 +1026,8 @@ type StmtResult = {
     "rows": Array<Array<Value>>,
     "affected_row_count": uint32,
     "last_insert_rowid": string | null,
+    "rows_read": uint64,
```

Comment:
Since `affected_row_count` is uint32, shouldn't `rows_read` and `rows_written` be uint32 as well?

Reply:
I am not following for why it should match the uint32 of affected row count? Is it just to save the 4 extra bytes?

Reply:
Just for consistency. if affected_row_count is uint32 and rows_written is uint64 then it is confusing. Does it mean max affected_row_count is 2^32 and max rows_written is 2^64? Maybe it's fine to have different types for those fields but it is confusing.

Reply:
lets solve this in a follow up. This feature is still hidden for now. Looks like there is both a uint64 and uint32 api for affected rows.